src/stories/AddressSearch.stories.tsx

```diff
@@ -15,12 +20,12 @@ export default {
     },
 };
 
-function Template(args) {
-    const [value, setValue] = useState('');
+function Template(args: AddressSearchProps) {
+    const [value, setValue] = useState<string | number | RenamedInputKeysProps | StreetValue>('');
     return (
         <AddressSearch
-            value={value}
-            onInputChange={({street}) => setValue(street)}
+            value={value as string}
```

Comment:
why do we need this assertion here ?

Reply:
This is needed because value can only be string but InputChange has multiple types that can be used.